spec/features/like_post_spec.rb

```diff
@@ -2,26 +2,27 @@
 require "support/features/likes_helpers"
 require "support/features/clearance_helpers"
 
-RSpec.feature "Likes", type: :feature do
+RSpec.feature "Like Post Functionality", type: :feature do
 
-  scenario "A post has a like button with like count" do
+  scenario "A post has a like button" do
     sign_in
     make_post
-    expect(page).to have_button "Like (0)"
+    expect(page).to have_button "Like"
   end
 
   scenario "A user can like a post" do
     sign_in
     make_post
-    click_button "Like (0)"
-    expect(page).to have_button("Like (1)", disabled: true)
+    click_button "Like"
+    visit '/'
+    expect(page).to have_content("1")
```

Comment:
This is liable to false-positive very easily if there are any `1`s on the page, which seems like there easily could be (in dates for instance).

I'd try querying a specific element, maybe targeting by class.